examples/mlj/binary.jl

```diff
@@ -6,24 +6,25 @@ using MLJBase
 using PythonCall
 
 # Initialize data
-cat_features = [0, 1]
-train_data = DataFrame([["a", "a", "c"], ["b", "b", "d"], [1, 4, 30], [4, 5, 40],
-                        [5, 6, 50], [6, 7, 60]], :auto)
-train_labels = [1, 1, -1]
-eval_data = DataFrame([["a", "a"], ["b", "d"], [2, 1], [4, 4], [6, 50], [8, 60]], :auto)
+train_data = DataFrame([coerce(["a", "a", "c"], Multiclass),
+                        coerce(["b", "b", "d"], Multiclass),
+                        coerce([0, 0, 1], OrderedFactor), [4, 5, 40], [5, 6, 50],
+                        [6, 7, 60]], :auto)
+train_labels = coerce([1, 1, -1], OrderedFactor)
+eval_data = DataFrame([coerce(["a", "a"], Multiclass), coerce(["b", "d"], Multiclass),
+                       coerce([0, 0], OrderedFactor), [4, 4], [6, 50], [8, 60]], :auto)
 
 # Initialize CatBoostClassifier
-model = CatBoostClassifier(; iterations=2, learning_rate=1, depth=2,
-                           cat_features=cat_features)
+model = CatBoostClassifier(; iterations=2, learning_rate=1, depth=2)
 mach = machine(model, train_data, train_labels)
 
 # Fit model
 MLJBase.fit!(mach)
 
 # Get predicted classes
-preds_class = MLJBase.predict(mach, eval_data)
+preds_class = MLJBase.predict_mean(mach, eval_data)
```

Comment:
Do you mean `predict_mode` here? This is a classifier.

Reply:
Yes it should. I have made this change.